docs/CHANGELOG.asciidoc

```diff
@@ -13,12 +13,553 @@
 
 This section summarizes the changes in each release.
 
+* <<release-notes-7.0.0-rc1>>
 * <<release-notes-7.0.0-beta1>>
 * <<release-notes-7.0.0-alpha2>>
 * <<release-notes-7.0.0-alpha1>>
 
 --
 
+
+[[release-notes-7.0.0-rc1]]
+== {kib} 7.0.0-rc1
+
+coming[7.0.0-rc1]
+
+[float]
+[[breaking-7.0.0-rc1]]
+=== Breaking changes
+
+For more details about breaking changes in this release, see 
+<<breaking-changes-7.0, Breaking changes in 7.0>>.
+
+Discover::
+* Does not apply `query:queryString:options` to `query_string` filters {pull}15640[#15640]
+* Removes `default_field` from `query:queryString:options` {pull}18966[#18966]
+Design::
+* Removes references to K6 EUI, using the default UI theme in its place {pull}29152[#29152]
+Geo::
+* Removes the deprecated `regionmap` and `tilemap` kibana.yml configurations. 
+Removes the backwards compatibility logic that checked for duplicative settings 
+`map.regionmap` and `map.tilemap`. {pull}28609[#28609]
+Monitoring::
+* Refactors index patterns from configurable to constants {pull}29528[#29528]
+* Removes `node_resolver` setting {pull}21181[#21181]
+Management::
+* Adds `configPrefix` to Index Management, License Management, Rollup Jobs, and 
+Upgrade Assistant {pull}30149[#30149]
+* Removes nested table splits from table visualization {pull}26057[#26057]
+* Standardizes use of `index_pattern` in UI code and URLs, replacing `indices` 
+and `index`  {pull}29151[#29151] and {pull}29481[#29481]
+* Removes app-specific dark mode options and replaces with a global dark mode uiSetting {pull}29017[#29017]
+Operations::
+* Defers the optimize step in the plugin installer to server start {pull}26983[#26983]
+* Removes tribe node support {pull}16397[#16397]
+* Creates separate startup scripts for development and production {pull}13806[#13806]
+* Sets default port based on protocol  {pull}21564[#21564]
+* Removes deprecated `/shorten` API {pull}21861[#21861]
+* Plugin installer defers optimization step until server start {pull}26983[#26983]
```

Comment:
Duplicate with `Defers the optimize step in the plugin installer to server start {pull}26983[#26983]`